test/sparrow_test.exs

```diff
@@ -37,20 +37,28 @@ defmodule SparrowTest do
   test "Sparrow starts correctly", context do
     # DON'T COPY THIS TOKEN GETTER
     with_mock Sparrow.FCM.V1, [:passthrough],
-      get_token_based_authentication: fn ->
+      get_token_based_authentication: fn _ ->
         getter = fn ->
           {"authorization", "bearer dummy_token"}
         end
 
         Sparrow.H2Worker.Authentication.TokenBased.new(getter)
       end do
+
       fcm = [
         [
           path_to_json: "sparrow_token.json",
           endpoint: "localhost",
           port: context[:port],
           tags: [:yippee_ki_yay],
           worker_num: 3
+        ],
+        [
+          path_to_json: "sparrow_token2.json",
+          endpoint: "localhost",
+          port: context[:port],
+          tags: [:yippee_ki_yay],
```

Comment:
add more tags, and different endpoint

Reply:
Why should I change the endpoint?

Reply:
to see if it works with any other endpoint than localhost